assets/js/components/Dashboard/PlaceRow.tsx

```diff
@@ -151,6 +151,25 @@ const PlaceRow = (props: PlaceRowProps): JSX.Element => {
     )?.inlineMap;
   };
 
+  const formatStationName = (stationName: string) => {
+    let isTerminalStop = false;
+    if (props.filteredLine && props.filteredLine !== null) {
```

Comment:
```suggestion
    if (props.filteredLine) {
```
^ this verifies `filteredLine` exists _and_ is truthy